snippets/functions.cson

```diff
@@ -24,6 +16,41 @@
     """
 
   #
+  # Ember Data 1.0.0-beta
+  #
+
+  'Ember Data: Find':
```

Comment:
What do you think of the name @seifsallam? I prefixed all of the Ember Data things to make it more obvious.